content/blog/living-off-the-ipad-as-an-engineer/index.md

```diff
@@ -0,0 +1,110 @@
+
+![Photo by [Ernest Ojeh](https://unsplash.com/@namzo?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/s/photos/magic-keyboard?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)](cover.jpeg)
+
+## External Monitor
+
+ I was lucky enough to have an [LG Ultrafine 4K](https://www.apple.com/shop/product/HMUA2VC/A/lg-ultrafine-4k-display?fnode=01ed6570fcad8a82974ba2365bf1eaa7d05d2098f328a409870e96ca8209d73125b43501b2178d7e0da5e2a872812d90d869256e6fac4e035ad1db81d91c5dca98f07b0806b261899f5ca87dcdf5f2b3bdd2a569c5a005cfba29a467eb8ce95e6d29ec0c8c4548ba56c842fb8e09c74a) display in my possession for use with the MacBook Pro. These fancy displays designed hand-in-hand with Apple are both compatible with Thunderbolt devices like the MacBook and with standard USB-C devices like the iPad. However, you can’t use the same cable, so if you are to buy this one, make sure you’re using the one with an iPad label on it before returning it out of frustration!
+
+I believe it’s common knowledge that you can’t just work out of a laptop form-factor all day without destroying your neck. Here the cheapest option would be to go with one of the pretty cool arm mount specifically designed to put the iPad right in front of your eyes or just a pile of books. This allows the expensive pixels to get their deserved amount of attention while also enabling instant video calling.
+
+While an external monitor is really comfortable, one thing to note is the lack of *full* external monitor support with iPadOS at the time of writing (version 14.4). Connecting a USB-C display like the Ultrafine essentially triggers an AirPlay mirroring of the iPad’s and is therefore really not as satisfying as a standard laptop+display setup, with the extended desktop for greater multitasking. Depending on its size and resolution, you’ll most likely end up with black bars around the mirrored video flux. Annoying even though you do get used to it. But they are ways around it.
+
+The workaround is indeed in the ability of iOS apps to specify how external monitor should be used. For instance, [Netflix](http://netflix.com) uses the iPad’s display for media controls while broadcasting the content onto the monitor. Luma Fusion has a mode for the video editor to stay on the iPad’s while live previewing on the monitor in full-screen.
+
+And in a clever workaround, a popular app called [Shiftscreen](http://shiftscreen.app) leverages these APIs to enable side-by-side web multitasking on the full external monitor, pretty convenient since nowadays a lot of work is actually happening within web apps like Google Docs or JIRA. On top of that, the app is still able to project the side-by-side browser on the external monitor even in Split Mode, therefore enabling another iOS app to be interacted with, such as a todo-list or direct messaging.
```

Comment:
And in a clever workaround, a popular app called [Shiftscreen](http://shiftscreen.app) leverages these APIs to enable side-by-side web multitasking on the full external monitor, which is pretty convenient since nowadays a lot of work is actually happening within web apps like Google Docs or JIRA. On top of that, the app is still able to project the side-by-side browser on the external monitor even when it's in Split Mode, therefore enabling another iOS app to be interacted with, such as a todo-list or direct messaging.